src/main/java/seedu/smartnus/model/choice/Choice.java

```diff
@@ -6,17 +6,19 @@
 
 import java.util.Collections;
 import java.util.HashSet;
+import java.util.Objects;
 import java.util.Set;
 
 public class Choice {
     public static final String MESSAGE_CONSTRAINTS = "Choices can take any values, and it should not be blank";
     public static final String MESSAGE_KEYWORD_CONSTRAINTS =
-            "Keywords can take any values, and they should not be blank";
+            "Keywords should not be blank and must contain alphanumeric characters.";
```

Comment:
Should this be "must contain **some** alphanumeric characters" instead?

Reply:
Hmmm because of the way I parse the keywords (within an answer string), I allow users to specify keywords like this `ans/ These are my keywords k/abc, k/def and k/jkl.` (with a punctuation mark connected) so I think I will display the message as such.